### PR TITLE
feat: Add redis_exporter role

### PR DESCRIPTION
**Pull request by anviar**
Add new role: **redis_exporter**

### CODE REVIEW

**Comment by anviar** [comment]
@SuperQ bot continue to add wrong label to this PR 🤷🏻‍♂️

**Comment by SuperQ** [comment]
Yea, very strange.

File: roles/redis_exporter/tasks/configure.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- name: Create redis_exporter config directory
+  ansible.builtin.file:
+    path: "/etc/redis_exporter"
+    state: directory
+    owner: root
+    group: root
+    mode: u+rwX,g+rwX,o=rX
+
+- name: Copy the passwords file
+  ansible.builtin.template:
```
Comment (SuperQ) [inline roles/redis_exporter/tasks/configure.yml:20]: I'm thinking maybe we should do this with `copy`.

```suggestion
  ansible.builtin.copy:
    content: "{{ redis_exporter_passwords | to_json(indent=2, sort_keys=True) }}"
```

Comment (anviar) [reply roles/redis_exporter/tasks/configure.yml]: nice trick, thank you. Update pushed.

**Review by SuperQ — APPROVED** [review]

**Pull request merged**